packages/json-deserializer/src/JsonDeserializer.ts

```diff
@@ -49,11 +49,15 @@ const isJsonMimeType = (event: APIGatewayProxyEvent) => {
 
   const mimeParts = contentTypeHeader.split("/");
 
+  if (mimeParts[1].includes(";")) {
+    mimeParts[1] = mimeParts[1].split(";")[0];
+  }
+
```

Comment:
I think this could be simplified to

```suggestion
  const MIME_TYPE_REGEX = /(.+)\/(.+);.+/;
  const mimeParts = MIME_TYPE_REGEX.exec(contentTypeHeader).slice(1);
```

Reply:
Why not:

Reply:
I check https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types and there are only two options allowed with `json`, so what about this:
```
const MIME_TYPE_REGEX = /application\/json|application\/ld\+json/;
if (!contentTypeHeader.match(MIME_TYPE_REGEX)) {
  return false;
}
```

Reply:
Good point. Maybe the full function could just be
```ts
const isJsonMimeType = (event: APIGatewayProxyEvent) => {
  const { headers } = event;
  const contentTypeHeader =
    headers?.["Content-Type"] ?? headers?.["content-type"] ?? '';
  return contentTypeHeader.startsWith("application/json") || contentTypeHeader.startsWith("application/jd+json");
}
```